decompiler/config/jak2/all-types.gc

```diff
@@ -11156,7 +11159,13 @@
    (giftag      generic-gif-tag :inline  :offset-assert 64)
    (adnops      uint32         2        :offset-assert 80) ;; guess
    (num-strips  uint32                   :offset 76)
+   (unknown-gcf-jki1n2ik1jn2  uint32                   :offset 80)
+   (unknown-gcf-dk12n1d  uint32                   :offset 84)
+   (unknown-gcf-kj2n423  uint32                   :offset 88)
    (num-dps     uint32                   :offset 92)
+   (unknown-gcf-kjn345  uint32                   :offset 96)
+   (unknown-gcf-asdas  uint32                   :offset 100)
+   (unknown-gcf-asdkh  uint32                   :offset 104)
```

Comment:
I think if you change `adnops` to `(adnops gs-adcmd 2 :inline)`, you can get rid of these unknowns.

Reply:
Also I am very curious to see what in ambient/speech/bigmap actually touches this...

Reply:
I found this in `lightning` as well, and the unknowns were my one reason for not finishing the file, so maybe i can now!